theme/themes/eea/extras/footer.variables

```diff
@@ -2,108 +2,80 @@
-@mobileThemeSitesPadding            : 0;
-@computerThemeSitesPadding          : 0rem 0 2rem;
-
-/* Theme Sites Header */
-@headerFontWeight                   : @bold;
-@mobileHeaderMarginBottom           : 1.25rem;
-@tabletHeaderMarginBottom           : 2.5rem;
-@computerHeaderMarginBottom         : 3.75rem;
-@mobileHeaderFontSize               : 0.688rem;
-@tabletHeaderFontSize               : 1.25rem;
-@computerHeaderFontSize             : 2.25rem;
+@mobileThemeSitesPadding            : @space-2 0 @space-8;
+@computerThemeSitesPadding          : @space-4 0 @space-8;
 
 /* Subfooter with logo and contact info */
-@mobileItemMarginTop                : 1rem;
-@tabletItemMarginTop                : 2rem;
-@computerItemMarginTop              : 4rem;
+@mobileItemMarginTop                : @space-4;
+@tabletItemMarginTop                : @space-8;
+@computerItemMarginTop              : @space-18;
 
 /* Subfooter Item Header */
-@itemHeaderFontWeight               : @bold;
-@mobileItemHeaderMarginBottom       : 0.625rem;
-@mobileItemHeaderFontSize           : 0.688rem;
-@tabletItemHeaderMarginBottom       : 0.938rem;
-@tabletItemHeaderFontSize           : 1.375rem;
-@computerItemHeaderMarginBottom     : 1.563rem;
-@computerItemHeaderFontSize         : 2.25rem;
+@mobileItemHeaderMarginBottom       : @space-5;
+@computerItemHeaderMarginBottom     : @space-12;
+@computerLogoHeaderMarginLeft       : -50px;
 
 /* Site Logo */
-@tabletSiteLogoWidth                : 192px;
-@computerSiteLogoWidth              : 283px;
+@tabletSiteLogoWidth                : 200px;
+@computerSiteLogoWidth              : 250px;
 @tabletSiteLogoDisplay              : inline-block;
 
-/* Site Logo Description */
-@mobileDescriptionMargin             : 5px 0 0 10px;
-@computerDescriptionMargin           : 5px 0 0 25px;
-@mobileDescriptionFontSize           : 0.5rem;
-@tabletDescriptionFontSize           : 1rem;
-@mobileDescriptionLineHeight         : 0.563rem;
-@tabletDescriptionLineHeight         : 1.188rem;
-
 /* EIONET Logo */
-@mobileEionetMarginTop: .5rem;
-@computerEionetMarginTop: 2rem;
-
-@mobileEionetLogoWidth               : 37px;
-@tabletEionetLogoWidth               : 65px;
-@computerEionetLogoWidth             : 80px;
-@mobileEionetLogoMarginLeft          : 0.75rem;
-@tabletEionetLogoMarginLeft          : 0.625rem;
-@computerEionetLogoMarginLeft        : 1.563rem;
-@mobileEionetLogoMarginRight         : 0.5rem;
-@computerEionetLogoMarginRight       : 1.25rem;
-
-/* eIONET Logo Description */
-@eionetDescriptionFontWeight         : @bold;
-@mobileEionetDescriptionPaddingTop   : 0.125rem;
-@tabletEionetDescriptionPaddingTop   : 0.313rem;
-@mobileEionetDescriptionFontSize     : 0.5rem;
-@tabetEionetDescriptionFontSize      : 1rem;
-@computerEionetDescriptionFontSize   : 1.063rem;
-@mobileEionetDescriptionLineHeight   : 0.625rem;
-@tabletEionetDescriptionLineHeight   : 1.5rem;
-@computerEionetDescriptionLineHeight : 1.8rem;
+@tabletEionetLogoWidth               : 225px;
```

Comment:
@g-stamatis for Antonis I would have liked to have the logos affected only by the column size and not have to use special width values, perhaps if more crisper logo's are outputted this could be removed in the future